includes/newspack-listings-utils.php

```diff
@@ -107,7 +107,7 @@ function get_data_from_blocks( $blocks, $source ) {
 		}
 
 		// If the source has 'single' specified, only get data from the first found block instance.
-		if ( $source['single'] ) {
+		if ( isset( $source['single'] ) && ! empty( $source['single'] ) ) {
```

Comment:
I think you only need `! empty()` here - the `isset` check is unnecessary.

Reply:
Confirmed and updated in b8a76e0. Guess I wanted to be *extra* certain :)